server/storage/rethinkdb_models.go

```diff
@@ -6,9 +6,10 @@ import (
 
 // These consts are the index names we've defined for RethinkDB
 const (
-	rdbSha256Idx        = "sha256"
-	rdbGunRoleIdx       = "gun_role"
-	rdbGunRoleSha256Idx = "gun_role_sha256"
+	rdbSha256Idx         = "sha256"
+	rdbGunRoleIdx        = "gun_role"
+	rdbGunRoleSha256Idx  = "gun_role_sha256"
+	rdbGunRoleVersionIdx = "gun_role_version"
```

Comment:
Fixed - just didn't notice that was already the primary key and didn't run the rethink tests locally.